app/Repositories/UserList/UserListRepository.php

```diff
@@ -28,6 +28,19 @@ public function findUserLists(int $userId) : Collection
         return UserList::where('user_id', $userId)->get();
     }
 
+    public function findUserListsWithPlaces(int $userId) : Collection
```

Comment:
This is not very good mix. 
```
->with([
                    'places',
                    'places.localization',
                    'places.city' => function ($query) {
                        $query->select(['id', 'name']);
                    },
                ])
```
UserList::with('places')->where('user_id', $userId)->get() is ok here too.